cmd/talosctl/cmd/talos/pcap.go

```diff
@@ -108,6 +108,11 @@ e.g. by excluding packets with the port 50000.
 				if err != nil {
 					return err
 				}
+				defer func() {
```

Comment:
this is totally minor, as the file will be closed anyway as the program terminates

but assignment to `err` won't work here

so better `defer out.Close() //nolint:errcheck` simply

Reply:
Another way is to print the error to `os.Stderr` which should be available here.